Dockerfile

```diff
@@ -72,6 +72,28 @@ RUN unzip awscliv2.zip
 # Install the bins to the /aws/bin dir so the final image build copy is easier
 RUN ./aws/install -b /aws/bin
 
+## Attach ocm binary into the image
+# Setting ENV variables
+ENV OCM_URL="https://github.com/openshift-online/ocm-cli/releases/download/v0.1.67/ocm-linux-amd64"
+ENV OCM_SHA256_URL="https://github.com/openshift-online/ocm-cli/releases/download/v0.1.67/ocm-linux-amd64.sha256"
```

Comment:
As the other bin, we should write the sha256 value here instead of get them from github.
And it will be a bit easier when we do the check next step.

Reply:
oh, actually, I  just referred the way used for getting shar256 for `oc` binary in this dockerfile and thought may be a convention? `RUN curl -sSLf ${OC_URL}/sha256sum.txt -o sha256sum.txt`

Yeah, it might be easier to hard code the value. Thanks